### PR TITLE
Resize browser width with a hook

### PR DESCRIPTION
**Pull request by mayorova**
**What this PR does / why we need it**:

Hopefully fixes random cucumber failures because an element can't be found/interacted with. This is caused by sidebar covering some elements, because the screen is too narrow.

**Which issue(s) this PR fixes** 

not reported

**Verification steps** 

All cucumbers must pass, all the time. 

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by mayorova** [comment]
> How is this supposed to help?
> 
> I think having a resize step is generally useful as it may allow more complex scenarios. But I'm fine as long as you are fine.

Well, the problem with the previous approach was that the window was made smaller, but never returned to the previous size.
We could add another step to return the size back, but I think it's error-prone:
1. you need to ensure you store the previous sizes (I didn't manage to find a way to get the configuration of the driver at runtime).
2. you also need to make sure you always include the "resize back" step, which is manual.

With `Before` and `After` hooks I think it's less error-prone.

**Comment by akostadinov** [comment]
Doesn't restart of the browser restore the original size? I think browser is restarted between scenarios.

**Comment by mayorova** [comment]
> Doesn't restart of the browser restore the original size? I think browser is restarted between scenarios.

I believe (although, I'm not 100% sure) the browser is not restarted between scenarios. What does happen is that the session is reset after each scenario, i.e. cookies are deleted and browser is redirected to a blank page: https://github.com/teamcapybara/capybara/blob/3.40.0/lib/capybara/cucumber.rb#L10-L12

> It maybe makes sense to set the browser to a fixed width and height in the beginning of every scenario to avoid environment differences to cause it to be different size between different executors. I remember doing so in the past. Dunno how it decides about it's initial size presently.

We're setting the window size through arguments passed to the chrome driver: https://github.com/3scale/porta/blob/f291877f07734754dfa46ec7451ab5c96b1fff0c/features/support/capybara.rb#L106

I don't think we need to resize the browser before each scenario, most of our scenarios (I guess ALL of them, except that single quickstarts button test) are perfectly OK with te default size. For a rear case where this resizing is needed, I thought that hooks for a specific scenario tag will do the trick.

**Review by jlledom — APPROVED** [review]
If this works, LGTM

File: features/support/hooks.rb [hunk 1]
```diff
@@ -39,6 +39,15 @@
   end
 end
 
+Before '@narrow-screen' do
+  @browser_width, @browser_height = page.driver.browser.manage.window.size.to_a
```
Comment (akostadinov) [inline features/support/hooks.rb:43]: btw ideally we should set these values as constants somewhere. But anyway, not super important.

**Review by akostadinov — APPROVED** [review]

**Pull request merged**